### PR TITLE
update small tag colors dark mode- RSC-1727

### PR DESCRIPTION
**Pull request by EmilyBermudez88**
https://sonatype.atlassian.net/browse/RSC-1727

Do we need to document these colors variables? If so, would they live on the CSS variables page, or perhaps a chart like the selectable colors page, but live on the NxSmallTag page?

### CODE REVIEW

File: lib/src/components/NxSmallTag/NxSmallTag.scss [hunk 1]
```diff
@@ -33,58 +35,67 @@
     padding: 0 var(--nx-spacing-2x);
     white-space: nowrap;
   }
+}
 
-  &--blue {
-    .nx-small-tag__text {
-      background-color: var(--nx-swatch-blue-30);
-    }
+.nx-small-tag--blue {
```
Comment (rpokorny) [inline lib/src/components/NxSmallTag/NxSmallTag.scss:40]: Why'd you change the syntax here?

**Comment by rpokorny** [comment]
> a chart like the selectable colors page, but live on the NxSmallTag page?

Let's do that ^

File: lib/src/components/NxSmallTag/NxSmallTag.scss (hunk 1 above)
Comment (EmilyBermudez88) [reply lib/src/components/NxSmallTag/NxSmallTag.scss]: > Why'd you change the syntax here?

reverted to previous syntax: [7b2ffb4](https://github.com/sonatype/sonatype-react-shared-components/pull/1152/commits/7b2ffb42b20dd55b7698f716bdb96bc1cfade1c9)

**Comment by EmilyBermudez88** [comment]
>> a chart like the selectable colors page, but live on the NxSmallTag page?

>Let's do that ^

Added in the color table: [dd1706c](https://github.com/sonatype/sonatype-react-shared-components/pull/1152/commits/dd1706c33731a1cfa43d953c6c838885d2b69ade)

**Comment by EmilyBermudez88** [comment]
>> a chart like the selectable colors page, but live on the NxSmallTag page?
>>Let's do that ^

>Added in the color table: [dd1706c](https://github.com/sonatype/sonatype-react-shared-components/pull/1152/commits/dd1706c33731a1cfa43d953c6c838885d2b69ade)

As discussed, removed the color table as it didn't add anything significant to the page: [123ec55](https://github.com/sonatype/sonatype-react-shared-components/pull/1152/commits/123ec55a8125e6b07df485d26cc15667205e6510)

**Review by rpokorny — APPROVED** [review]

**Pull request merged**